src/docs/drizzle/react/react-integration.md

```diff
@@ -4,145 +4,158 @@ layout: docs.hbs
-   ```javascript
-   // For convenience
-   constructor(props, context) {
-     super(props)
-
-     this.contracts = context.drizzle.contracts
-   }
-
-   // If Drizzle is initialized (and therefore web3, accounts and contracts), fetch data.
-   // This will update automatically when the contract state is altered.
-   var storedData = this.props.drizzleStatus.initialized ? this.contracts.SimpleStorage.methods.storedData.data() : 'Loading...'
-   ```
-
-   The contract instance has all of its standard web3 properties and methods. For example, sending a transaction is done as normal:
-   ```javascript
-   this.contracts.SimpleStorage.methods.set(this.state.storageAmount).send()
    ```
 
-## Recipe: Loading Component
-
-The following wrapper and component will detect when your dapp isn't ready and allow you to display the appropriate status or course of action:
-
-`LoadingContainer.js`
-
-```javascript
-import Loading from './Loading.js'
-import { drizzleConnect } from 'drizzle-react'
-
-// May still need this even with data function to refresh component on updates for this contract.
-const mapStateToProps = state => {
-  return {
-    drizzleStatus: state.drizzleStatus,
-    web3: state.web3
-  }
-}
-
-const LoadingContainer = drizzleConnect(Loading, mapStateToProps);
-
-export default LoadingContainer
-```
-
-`Loading.js`
-
-```javascript
-import React, { Component, Children } from 'react'
-
-class Loading extends Component {
-  constructor(props, context) {
-    super(props)
-  }
-
-  render() {
-    if (this.props.web3.status === 'failed')
-    {
-      return(
-        // Display a web3 warning.
-        <main>
-          <h1>⚠️</h1>
-          <p>This browser has no connection to the Ethereum network. Please use the Chrome/FireFox extension MetaMask, or dedicated Ethereum browsers Mist or Parity.</p>
-        </main>
-      )
+1. Import `Drizzle` and `generateStore`.
+    ```javascript
+    import { Drizzle, generateStore } from "@drizzle/store";
+    ```
+
+1. Create a new `drizzle` instance with the `drizzleOptions` object and the generated store.
+    ```javascript
+    const drizzleStore = generateStore(drizzleOptions);
+    const drizzle = new Drizzle(drizzleOptions, drizzleStore)
+    ```
+
+1. Pass the `drizzle` object to the `DrizzleContext.Provider` component.
+    ```javascript
+    <DrizzleContext.Provider drizzle={drizzle}>
+    </DrizzleContext.Provider>
+    ```
+
+1. Use `DrizzleContext.Consumer` to consume the drizzle context and pass `drizzle` and `drizzleState` to your component. Drizzle also provides prebuilt components via the `@drizzle/react-components`.
+
+    **Note**: We have to check that Drizzle is initialized before fetching data. The `initialized` variable returns the drizzle store's initialization status.
+
+    ```javascript
+    <DrizzleContext.Provider drizzle={drizzle}>
+            <DrizzleContext.Consumer>
```

Comment:
Formatting issue w/ the indentation here. We want to stick to 2 spaces for each indentation if possible.